Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -1694,6 +1705,19 @@ static void webkit_web_view_class_init(WebKitWebViewClass* webViewClass)
         nullptr,
         static_cast<GParamFlags>(WEBKIT_PARAM_READWRITE | G_PARAM_CONSTRUCT_ONLY));
 
+    /**
+     * WebKitWebView::theme-color:
+     *
+     * The theme color of the WebView's current page.
+     *
+     * Since: TBD
```

Comment:
The next major release series will be 2.48, therefore:

```suggestion
     * Since: 2.48
```